README.md

```diff
@@ -111,3 +111,7 @@ npm install @googlemaps/google-maps-services-js
 ## Install Font Awesome packages
 
 `npm install --save @fortawesome/react-fontawesome @fortawesome/free-solid-svg-icons`
+
+## Install react tabs
```

Comment:
Ah I understand now, because it wasn't in the package for me, so i thought other users need to know install tabs , but i've already done that for them and it's in the package.json.